### DISCUSSION TITLE
Is there a way playing background music during presentation on selected slides or selected time or selected songs?

### DISCUSSION THREAD

**Discussion by zcatav** [quarto-dev/quarto-cli#5136] [Q&A] [answered]
Hi,
Is there a way playing background music during presentation on selected slides or selected time or selected songs?
Thanks for your attention.

Ps: I'm using revealjs as slide presentation.

**Comment by mcanouil** [comment, marked as answer]
You can use many ways to do that following any of available solutions on the Internet, see <https://duckduckgo.com/?q=music+html+background> for few of them.

You can also include a music reader, include iframe, _etc._
